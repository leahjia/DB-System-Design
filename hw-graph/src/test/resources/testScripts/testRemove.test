# University Map Node Removal testing file.
# Tests that a single node can be removed from the university map

# Remove a pair of nodes
RemoveNode UnivMap A
RemoveNode UnivMap B
