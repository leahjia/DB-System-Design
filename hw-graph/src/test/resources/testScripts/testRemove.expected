# University Map Node Removal testing file.
# Tests that a single node can be removed from the university map

# Remove a pair of nodes
removed node A from UnivMap
removed node B from UnivMap
